lib/libvfio-user.c

```diff
@@ -1426,6 +1453,12 @@ vfu_run_ctx(vfu_ctx_t *vfu_ctx)
             err = handle_request(vfu_ctx, msg);
             free_msg(vfu_ctx, msg);
             reqs_processed++;
+#ifdef DEBUG
```

Comment:
why isn't this at the end of vfu_device_quiesced() ?

Reply:
sure

Reply:
Hang on, it _is_ at the end of vfu_device_quiesced(), but that's not enough because the device might immediatelly quiesce as you say.

Reply:
this is vfu_run_ctx() !

Reply:
I mean `vfu_ctx->quiesced = false` is already at the end of `vfu_device_quiesced()`.

Reply:
sure, but that's fine, no? Doesn't harm anything to immediately mark it unquiesced there

Reply:
I'm asking for some comments on this, as it's not immediately obvious why we need the check in vfu_run_ctx() too. At least I - clearly - always forget this

Reply:
I've added a comment here: https://github.com/nutanix/libvfio-user/pull/647/commits/f7d2096ad94438a391244a056e7a8fccdf8cfa8a